privaterelay/templatetags/relay_tags.py

```diff
@@ -15,3 +15,14 @@ def remaining_free_aliases(aliases):
 def user_email_domain(user_profile):
     mail_domain = get_email_domain_from_settings()
     return "%s.%s" % (user_profile.subdomain, mail_domain)
+
+@register.simple_tag
+def message_in_fluent(message):
```

Comment:
Let's keep in mind to remove this later when we get all messages into Fluent, yes?